src/Ast/PhpDoc/MethodTagValueNode.php

```diff
@@ -20,17 +20,21 @@ class MethodTagValueNode implements PhpDocTagValueNode
 	/** @var string */
 	public $methodName;
 
+	/** @var MethodTagValueGenericNode[] */
+	public $generics;
+
 	/** @var MethodTagValueParameterNode[] */
 	public $parameters;
 
 	/** @var string (may be empty) */
 	public $description;
 
-	public function __construct(bool $isStatic, ?TypeNode $returnType, string $methodName, array $parameters, string $description)
+	public function __construct(bool $isStatic, ?TypeNode $returnType, string $methodName, array $generics, array $parameters, string $description)
```

Comment:
This parameter needs to be added as last and needs to be optional, otherwise it's a BC break.